### ISSUE TITLE
The famous game "GTA5" crashed in the sandbox

### ISSUE THREAD

**Issue by Taeyeonzpx** [sandboxie-plus/Sandboxie#3612]
### Describe what you noticed and did

The game can start, but crashes 2 seconds after starting. It could be played normally before, but it will crash in the latest version of the sandbox
![屏幕截图 2024-02-08 115307](https://github.com/sandboxie-plus/Sandboxie/assets/89934966/4cd6a549-e6ad-428b-957f-25a0dd028f69)

### Expected behavior

I hope it starts normally

### Affected program

GTA5

### Download link

Not available

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Did the program or any related process close unexpectedly?

Yes, it did. I'm going to share the .dmp file(s) in a later comment.

### Crash dump

https://drive.google.com/file/d/1-ygQ9x8q-P0oZHejGZ0cVSrGfn4fZx2W/view?usp=sharing

### What version of Sandboxie are you running now?

plus 1.12.9

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I remember which one it is).

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

Not relevant to my request.

### What is your Windows edition and version?

WINDOWS11 PRO  64BIT

### In which Windows account you have this problem?

A local account (Standard user).

### Please mention any installed security software

Huorong

**Comment by Dyras** [comment]
Can you try downgrading a few versions and see which version worked?

**Comment by Taeyeonzpx** [comment]
> Can you try downgrading a few versions and see which version worked?

I finally tried it and found that the problem was with the game itself and not with the sandbox. The problem was finally solved. The game could start normally, but there was still a long-standing problem, which was that the page reported an error "SBIE2224 crash" when exiting the game. This is the log https://drive.google.com/file/d/1jGi7iXG5418IP8tvvk7BxZnORekTh3m0/view?usp=sharing
